modules/andes-core/client/src/main/java/org/wso2/andes/XaJmsSession.java

```diff
@@ -277,6 +278,43 @@ public void unsubscribe(String s) throws JMSException {
         session.unsubscribe(s);
     }
 
+    @Override
```

Comment:
Let's include javaDocs ? you could use {@inheritDoc} to refer to the docs included in the interface